primitives/consensus/babe/src/digest.rs

```diff
@@ -44,6 +44,8 @@ use sp_std::vec::Vec;
 pub enum BabePreDigest {
 	/// A primary VRF-based slot assignment.
 	Primary {
+		/// epoch threshold
```

Comment:
Probably breaks Kusama.

Reply:
I guess that means the predigest is stored in the db?

I could make it an `Option<u128>`

Reply:
It's stored in the block headers. There should be no need to change the format here.

Reply:
The threshold doesn't need to be passed in the digest, it is trivially calculated given `c` (a configuration constant), the authorities for the epoch and the authority index that we want to calculate the threshold for (check `claim_primary_slot`). We also don't need to expose this info through RPC which seems like the only reason this change was done.